pages/dkp/kommander/2.2/install/air-gapped/index.md

```diff
@@ -224,7 +224,7 @@ It may take a while to push all the images to your image registry, depending on
     ```bash
     kommander install --installer-config ./install.yaml \
```

Comment:
same here

Reply:
to avoid conflicts, I'll defer this to the other PRs being worked on to change these